src/css-angle-value.js

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//     You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//     See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(internal, scope) {
+
+  AngleType = {
+    DEG: "deg",
+    RAD: "rad",
+    GRAD: "grad",
+    TURN: "turn"
+  };
+
+  function isValidAngleUnit(unit) {
+    return internal.objects.any(AngleType, function(type) {
+      return unit == type;
+    });
+  }
+
+  function rad2deg(rad) {
```

Comment:
Done. These are what the internal methods in blink are called. No need to follow that though.